sentry/src/main/java/io/sentry/Scopes.java

```diff
@@ -230,6 +231,37 @@ private IScope buildLocalScope(
     return sentryId;
   }
 
+  @Override
+  public @NotNull SentryId captureFeedback(
+      final @NotNull Feedback feedback,
+      final @Nullable Hint hint,
+      final @Nullable ScopeCallback scopeCallback) {
+    SentryId sentryId = SentryId.EMPTY_ID;
+    if (!isEnabled()) {
+      getOptions()
+          .getLogger()
+          .log(
+              SentryLevel.WARNING,
+              "Instance is disabled and this 'captureFeedback' call is a no-op.");
+    } else if (feedback.getMessage().isEmpty()) {
+      getOptions()
+          .getLogger()
+          .log(SentryLevel.WARNING, "captureFeedback called with empty message.");
+    } else {
+      try {
+        final IScope localScope = buildLocalScope(getCombinedScopeView(), scopeCallback);
+
+        sentryId = getClient().captureFeedback(feedback, hint, localScope);
+      } catch (Throwable e) {
+        getOptions()
+            .getLogger()
+            .log(SentryLevel.ERROR, "Error while capturing feedback: " + feedback.getMessage(), e);
+      }
+    }
+    updateLastEventId(sentryId);
```

Comment:
Hm, do we do this on other platforms too? Not sure if feedback event_id is useful as lastEventId, wdyt?

Reply:
Yeah, we don't
Technically, the last event is the feedback, but i think we introduced this API mainly to attach the last event id to the feedbacks, so yeah, I can remove it